server/models/Profile.js

```diff
@@ -0,0 +1,45 @@
+const mongoose = require('mongoose');
+
+const profileSchema = new mongoose.Schema({
+    firstName: {
+        type: String,
+        required: true,
+        trim: true
+    },
+    lastName: {
+        type: String,
+        required: true,
+        trim: true
+    },
+    description: String,
+    gender: {
+        type: String,
+        required: true,
+        enum: ['Male', 'Female']
+    },
```

Comment:
Awesome to see `enum` here! Although, it might be best to give the users the option to select something along the lines of `other` (or even not disclose this info altogether). :^) 

A small improvement for providing a more flexible API would be to apply one of `lowercase/uppercase` (and/or `trim` here too!) to avoid casing problems 🙌🏼